### PR TITLE
Updated agent section in architecture - info about disabling capabilities

### PR DESCRIPTION
**Pull request by Razz4780**
Configuration doc does not contain info about `disabled_capabilities` field yet. Is it updated automatically?

### CODE REVIEW

**Comment by eyalb181** [comment]
Semi-automatically - please generate new configuration with medschool (see mirrord/medschool)

File: content/docs/overview/architecture/index.md [hunk 1]
```diff
@@ -31,6 +31,9 @@ mirrord-agent does **not** run as a privileged container in the cluster. However
 - `CAP_SYS_PTRACE` - required for reading target pod environment
 - `CAP_SYS_ADMIN` - required for joining target pod network namespace
 
+However, you can disable any subset of those in the [configuration]({{< ref "/docs/overview/configuration" >}} "configuration").
```
Comment (eyalb181) [inline content/docs/overview/architecture/index.md:34]: Mention the actual parameter name and link to it in the configuration if possible

File: content/docs/overview/configuration.md [hunk 2]
```diff
@@ -93,11 +93,11 @@ configuration file containing all fields.
       },
       "dns": false
     },
-    "capture_error_trace": false
   },
   "operator": true,
   "kubeconfig": "~/.kube/config",
-  "sip_binaries": "bash"
+  "sip_binaries": "bash",
```
Comment (eyalb181) [inline content/docs/overview/configuration.md:99]: I think `disabled_binaries` should have been added to this example. Medschool has some determinism issues - could you try running it again and see if anything's different?

Comment (Razz4780) [reply content/docs/overview/configuration.md:99]: I ran medshool a couple times, each time produces the same output

**Review by eyalb181 — APPROVED** [review]

**Pull request merged**